discovery/v1.js

```diff
@@ -545,6 +545,32 @@ DiscoveryV1.prototype.addJsonDocument = function(params, callback) {
  * @param {String} params.environment_id environment guid for the collection
  * @param {string} params.collection_id the guid of the collection
  * @param {string} params.document_id the guid of the document to update
+ * @param {string} params.document_id the guid of the document to update
+ * @param {Buffer|ReadableStream|Object} params.file a file to post (smaller than 50mb)
```

Comment:
This line should say 

     * @param {Object} params.file non-stringified JSON object